src/tools/dev/scripts/bv_support/bv_mili.sh

```diff
@@ -1028,8 +1028,15 @@ function build_mili
          extra_ac_flags="ac_cv_build=powerpc64le-unknown-linux-gnu"
     fi
 
+    # The 19.2 configure script does not play well with our fortran mac patch.
+    # We can use an older configure script that includes this patch already.
+    config_script=configure
+    if [[ "$OPSYS" == "Darwin" ]]; then
+        config_script=configure_15_1
```

Comment:
This configure_15_1 script is already part of the mili library?

Reply:
No, I added it to our tarball. It turns out that our previous tarball included an already patched version of the configure script, and that's what I've copied over (and renamed). I suppose we could also keep this separate from the library and add it after untarring... Any thoughts?

Reply:
I just wanted to be sure it would be available during use. You've committed the modified tarball already, or will do soon I presume.

Reply:
That's right. I updated our thirdparty libs repo this morning.